packages/site/src/data/vue/communities.ts

```diff
@@ -44,7 +44,7 @@ export const communities: Community<typeof communityTags[number]>[] = [
cXqCWjFAn16dN31GaZKEhYaesvSaibuNwRPDVwmTwTMJPQqrDclCOcK7g4gXLTZo2RsXIJF7lYI0IyfHpyEFHHzWj92YSzXKyg2KxGCIgXhzAnbgsTIqJRST1dkTA0IVCbLsanw7Z2vgZqjofWDLEmhCaupZ4fNtm2lBAhc/FFFam4sod6SM6Pmxr2RUFC0L0KEFfNQ1wyJt3jZtTGriFKuPlo/YglZk/LgHAAtH4yOR4A4s1P5BBWEqfzGbBZzsNAx9ENCcY/DGAUvtz5KY/Q2ny2fkTOKgbBFSjIODNnjQOn/t3TQZs6seayIY4DecklrCITRVai8kWmk0tDmwUqqzg8QtCedIS4PMNDDILxpQEz4phFKvZHPmFDKkIzDVSuTD0Ynxq4M+WkI1SHH6WNl2tHLIpSbO4cQCw44KJGWQ5iLvI8BMQnshGG0BZ0jpFhCD5joKKBrPsXROl3MhFCiCJAEHBiNcQ8JePtQRSlg9SiPJSLEIJQJ6Mfu7xqE7B5//ixwU6fcrj9UEpCMmr95a8jkpbUDM4Afb5PguHfoPWzclMnQjbOId0H/tsuQa8QR8/xk+BasY21BgitXq/h+XUh1IEweTsOGO3c26xoHR7ttiOquUDoAYnn1YUQPPQ6yfsxWKjz91GHR+/aUE2RMIdOSkKChJaVvO8u52GGi2GYIqaE+YjVED4uyEPUpwlrGnkbjWa6VnwyaBsfoLHnI1bT8YsI0RPv08cg1GeIGR4S7CzDpyeR8QHrTC6ilIReA76z5X/6yBuGBynixfcQpb2e59eOEL4yII44w5t0ADS82P17/Qh93+p5iW91roYj1jUyKw28eT8kwfTNP3zMQ4kIbUaYhfjVw7mufmoRdmJGnpGg1pbf8DodiQg1PqG+TJhc/zwqmjSOpCMs8PAGoe8DIsw1Qs7ZKfX20OtZ1z+3cKM0rBFhGQ9hTG1dQaDiL/Km/vIRlvGwB42jgYHKaYwzwo48hGU8tGBO1POuX3U58375CMt42IE+Z/X861es9WfHqXyEZTzsdCzcCfSSf6atP+soSQkzS2NGpZkr2eoGBIei+KdhWxiLo0bSEWqrEPpXb7ssUNMVueUFR/kIV/LQaiSICP8lnW8sRbl8hCt52OtZiDi/HExfGgHIR7iShz1A9AGRXQ9G0nCVlnC1PIT3cH2qhfaFN/p/AeGfauJh4uE2S/J+0gp0rKRSE67kIU5wYYyKLgbmzemUfIQFHprZedhhSxue9y8M1DJ5WBVh9nyP9YFMwoXSxRu21BamJUc+wvxxqRBh8ukjCZc3w+Uj5HtoFhBafnL9kejzroi7cdIR8ucWRYQ9L0mufwpSFyUlTCtNPmHrgc9OMcghhLLawWk/dkV9iTDvgMq6RQ5haIbd975l5SJ6WFE7V59HixNTeISWlB52f/GQLw/R6jV8HxBbQZqLpnyEuR7i2Xzo4fGvjXwP2S6T5fnJ59k4PJSPMN9DPF8xvN/aMAAxz0Or43s4CveuPnfhAGiNQOhJSJhfS4GwjVvzDU4yonpsNgU+DreSFDr7/63/DNOiboGE7OyDAkKL7fbDKLy75fs+EOb0mAoIBTzUNON1kYe+78GEEQK1u+WxvbWeNIQCHtpzRA4hLjNCoBLS2vItnHvIQyjkISD+ZqXJmIfYs6xO8nYynL5K2LRDHkKRPGSI/7Yam7k1Fc8SgskUzIm7rxJ8nVNoJPaQIfIHNw3P8pIH578k+FoiQlEPAfFJvoWAyGLY9//zXyylDXmiVNP6vH38ZUIuIsYl9AhIxo6XeA2J8tDR2kAY6BnrGHcINePZXqOobRRo/YR9jTYfB4KEmvFor6j1S0ioNbdJZiJmEM5drBMhdPPm6TRznyyL0DYe7nHKjaSEzuGYZJ0ik0UILj78PU3F1TgrqKVUcw4upuKEgLjzDdWmCkLqDLbHWf0ih/DbECvoFu3IwTDNmAPnEdqGsVMwDpeJMI4i5+zgKChBCC4yxJoQ2rbtNt2TCzTx1j5ZPiEgvuDPNHK1/gsQ+45GXXdwmpZTc/nMSg5hiriKjesndBwtarvuwSkuB7KLZIQIIRlXQ6yEsI23fmrujwOwMDQFCcHGF6vkYgWXybLL6mAOZZziNRXLO4F8D8HFX1PEUphrJ7Qp1eIYr3OFQIWuKEyIx85XGaUmxMsi47OYYqAiIlkaoRYRAuKHhvyE0PG1M7xvd6S5A8xFoW4xE7hYttqsndBB++LY6Ts0cl1jv7U0uCkmhHLzQXoPtfnNrPvgYty8xHJjpq1fhBBXGUvNiSsgnCuGjGy6u/us9bP7JOsihGyVsSaEDoXOv/vudDiatX4hDyEXGWItCG3ahuGNkZYbaP1ihLOF1BoQYut3XTeOAZGdQRSKEmrGk03hzl8hIcz105s9u8Zl2vpFCW3NeLYpOieukjBFpNj6L/G6X1PYQ3Dx5aZgLlYZpey+69AWI3uWiyUIZy7KTRhrEbuinuIIztgfsigN5/uHfOEG48s9EQsr9XD+1KB+FLnY+vGOEGFodr8UE6LYQmpxLlZKOFNM29Rxd7fHbFxjtjaaYsfNFlJrQHjmsGqze9kaYSIGooQzxBoQ4hOCoDGyWT+MUVsbwt+JIdaAEAWATox9UddbG2L3YkcZxnPW+HO3wi0ZCO0IbzFHMVIH2PpbgpVGY62fuz5lSeJhej9SfFyXNtgeB90vonmISl3MDVUpPNQcRMTWT/satP7ul1JHs1XG3CyUgxDhtPgstnF5ytifiEcpEy7B5cta/72+MrV4PiBtG3/8IfQAnSUZr61G5nlw8M6mHIAL4f27YM5Y9jDj5YvNnFL6WkJCKvbwlRsyjEevn2/e1d5vkgHiUJWudu9ffL6g/fC2Kn8URKaczLt2i8g2bPvGIxWleOzMHTmrEyoprUur1FIlJSUlJSUlJSUlJSUlJSUlJSUlJSUlJSUlJSUlJSUlJSUlJSUlJaX6639n+045UNCnTAAAAABJRU5ErkJggg==",
```

Comment:
Please remove this because you already committed these changes in your last PR.

Make sure to merge the latest changes from `main`.